applications/asset_tracker/src/gps_controller/gps_controller.c

```diff
@@ -102,7 +102,7 @@ static void stop(struct k_work *work)
 		LOG_ERR("Failed to disable GPS, error: %d", err);
 		return;
 	}
-	k_delayed_work_cancel(&start_work);
+	k_work_cancel_delayable(&start_work);
```

Comment:
What happens if this fails? Would this be simpler if we just had one work item and used a flag to determine whether to start or stop?

Reply:
considered it to be safe since they run on same queue